rest/utilities_testing.go

```diff
@@ -814,3 +1046,95 @@ func EnableBlipSyncLogs() {
 	base.EnableLogKey("Sync")
 	base.EnableLogKey("Sync+")
 }
+
+
+type AttachmentMap map[string]*DocAttachment
+
+type DocAttachment struct {
+	ContentType string `json:"content_type,omitempty"`
+	Digest      string `json:"digest,omitempty"`
+	Length      int    `json:"length,omitempty"`
+	Revpos      int    `json:"revpos,omitempty"`
+	Stub        bool   `json:"stub,omitempty"`
+	Data        []byte `json:-` // tell json marshal/unmarshal to ignore this field
+}
```

Comment:
This shouldn't be test-only - it should be defined (and used) by attachment.go.  If that's out of scope for this PR, can we at least move the struct to attachment.go, and include comments saying it should be incorporated the next time someone's making a change in attachment.go?

Reply:
Sounds good